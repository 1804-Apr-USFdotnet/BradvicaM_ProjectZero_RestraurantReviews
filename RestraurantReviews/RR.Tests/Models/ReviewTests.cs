using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using RR.Models;

namespace RR.Tests.Models
{
    [TestClass]
    public class ReviewTests
    {
        [TestMethod]
        [UseReporter(typeof(DiffReporter))]
        public void ToString_OnCall_DisplayProperly()
        {
            var review = new Review
            {
                Comment = "It was kinda meh.",
                Rating = 4.45,
                ReviewId = Guid.Empty
            };

            Approvals.Verify(review);
        }
    }
}
