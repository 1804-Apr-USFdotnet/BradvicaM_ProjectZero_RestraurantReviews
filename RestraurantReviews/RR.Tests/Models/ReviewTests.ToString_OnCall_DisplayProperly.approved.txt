
Id: 00000000-0000-0000-0000-000000000000
Rating: 4.45
Comment: It was kinda meh.
Name: 
